### PR TITLE
[KEYCLOAK-11289] Enable Pulp repos for OSBS builds

### PR DESCRIPTION
**Pull request by iankko**
Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@goldmann PTAL once got a chance

Thanks!, Jan

File: container.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+compose:
+  pulp_repos: true
+platforms:
+  only:
+  - x86_64
```
Comment (goldmann) [inline container.yaml:5]: Are you building only for x86_64?

**Comment by goldmann** [comment]
I would expect to have both defined here, `content_sets.yml` and `container.yaml`. Why this is not the case?

File: container.yaml (hunk 1 above)
Comment (iankko) [reply container.yaml:5]: Yes, only for x86_64 for now (ppc64, ppc64le) wasn't investigated so far AFAICT>

**Comment by iankko** [comment]
> I would expect to have both defined here, `content_sets.yml` and `container.yaml`. Why this is not the case?

These are in `image.yaml` directly under `content_sets:` clause. Here:

https://github.com/jboss-container-images/redhat-sso-7-image/blob/sso-cd-dev/image.yaml#L25

 Should I move them to separate file instead?

**Comment by goldmann** [comment]
> These are in image.yaml directly under content_sets: clause. Should I move them to separate file instead?

I don't see them TBH.

**Comment by iankko** [comment]
> > These are in image.yaml directly under content_sets: clause. Should I move them to separate file instead?
> 
> I don't see them TBH.

https://github.com/jboss-container-images/redhat-sso-7-image/blob/sso-cd-dev/image.yaml#L25

**Comment by goldmann** [comment]
Ah, ok, I was looking at the default branch, sorry. No, it's not required to move them. Content of container.yaml file can be embedded too, if you want.

**Review by goldmann — APPROVED** [review]

**Comment by iankko** [comment]
@goldmann Thanks, Marek!

**Pull request merged**